src/Command/CreateAdminCommand.php

```diff
@@ -20,6 +20,8 @@
 
 final class CreateAdminCommand extends Command
 {
+    protected static $defaultName = 'repman:create:admin';
+
```

Comment:
This will make command lazy, there was a warning when Telemetry service was injected and database was not created yet (requires messenger.transport.failed)